lochness/xnat/__init__.py

```diff
@@ -278,7 +294,9 @@ def sync_xnatpy(Lochness, subject, dry=False):
                         project=experiment.project,
                         subject=xnat_subject.label,
                         session=experiment.label,
-                        out_dir=tmpdirname
+                        out_dir=tmpdirname,
+                        xnat_username=keyring['USERNAME'],
+                        xnat_password=keyring['PASSWORD']
```

Comment:
Before, did we use an API token for real XNAT instead of username-password combo?

Reply:
Previously, with yaxil and pyxnat we authenticated with something like:

- yaxil:
    ```python
    auth = yaxil.XnatAuth(url=Keyring['URL'], username=Keyring['USERNAME'],
            password=Keyring['PASSWORD'])

    yaxil.download(auth, experiment.label,
        project=experiment.project,
        scan_ids=['ALL'], out_dir=dst,
        in_mem=False, attempts=3,
        out_format='native')
    ```
    
   - pyXNAT
    ```python
    session = xnat.connect(keyring['URL'],
        keyring['USERNAME'],
        keyring['PASSWORD'])
    ```
    
    Similarly, for dataorc we use the same username and password.